NOTICE

```diff
@@ -0,0 +1,11 @@
+Apache Pekko
+Copyright 2022, 2023 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (https://www.apache.org/).
```

Comment:
Is it worth mentioning https://pekko.apache.org/ ?

Reply:
NOTICE files are generally added for ASF projects. I've never seen them include project specific URLs. An example is the Apache Spark NOTICE file.

https://github.com/apache/spark/blob/master/NOTICE